vector/src/main/java/im/vector/app/core/di/SingletonModule.kt

```diff
@@ -125,12 +125,8 @@ object VectorStaticModule {
                 applicationFlavor = BuildConfig.FLAVOR_DESCRIPTION,
                 roomDisplayNameFallbackProvider = vectorRoomDisplayNameFallbackProvider,
                 threadMessagesEnabledDefault = vectorPreferences.areThreadMessagesEnabled(),
-<<<<<<< HEAD
-                presenceSyncEnabled = BuildConfig.PRESENCE_SYNC_ENABLED,
```

Comment:
You keep PRESENCE_SYNC_ENABLED config in gradle file but you don't use it here anymore. 
Why ?

Reply:
they don't use it in Element anymore.
I will delete it in the gradle file